docs/guidelines/content-guidelines/02-style-and-terminology.md

```diff
@@ -12,24 +12,24 @@ These are the generally accepted grammar rules for writing Kyma documentation.
 
 Use active voice whenever possible. Active voice is clear, concise, and it avoids misinterpretation. It is also easier for non-native speakers to understand. Passive voice is indirect, uses more words, and can be misleading because it reverses the logical order of events.
 
-✅ The endpoint path includes your service name.   
+✅ The endpoint path includes your service name.  
 ⛔️ Your service name is to be included in the endpoint path.
 
 ### Voice and tone
 
-There are different tones for different types of technical documentation. The documentation can range from instructional to somewhat conversational, but always with the goal of helping users understand how to use the product for practical purposes and, in blogs and release notes, also helping business users understand changes. While writing Kyma documentation, use semi-formal style and imperative mood. The imperative mood tells the reader directly to do something. Use the imperative mood to write instructional documentation such as procedures and tutorials. Other moods can imply optional behavior.
+There are different tones for different types of technical documentation. The documentation can range from instructional to somewhat conversational, but always with the goal of helping users understand how to use the product for practical purposes and, in blogs and release notes, also helping business users understand changes.
+While writing Kyma documentation, use semi-formal style and imperative mood. The imperative mood tells the reader directly to do something. Use the imperative mood to write instructional documentation such as procedures and tutorials. Other moods can imply optional behavior.
 
-> **NOTE:** Avoid using unnecessary words such as "please" or "remember."
+> **NOTE:** Avoid click-level instructions. Avoid using unnecessary words such as "please" or "remember."
 
 ✅ Click **Add**.  
-✅ Click the **Add** button.  
-⛔️ Please, click **Add**.    
+⛔️ Click the **Add** button.  
```

Comment:
> ⛔️ Click the **Add** button.

Why are we introducing this change? 
Although I'm all for brevity and conciseness, I do believe sometimes the "fuller" version sounds better, especially if using just the name might be ambiguous or confusing, or if there are more UI elements with the same or similar text, or the text from the button is repeated in the sentence.

I'd strongly suggest changing this into a suggestion/recommendation rather than a strict rule and leaving the decision to the writer, who knows the context best.

Reply:
I agree, if the UI is ambiguous we need to be more specific. But how often does that happen?
99% of the time, people write “Click the **Add** button.” when “Add your XY…” would be perfectly sufficient.

Click-level instructions add dependency on the UI strings, creating maintenance effort to make sure that the UI still says “Add” and didn’t change in the meantime.

Reply:
How about "Avoid click-level instructions unless the UI is confusing. Readers of Kyma documentation are assumed to be fairly tech-savvy."

Reply:
As a user, I liker (and prefer) click-level instructions. I don't think it's now always about being tech savvy. Imagine if you went to an Azure or AWS dashboard and had no click-level instruction. I'd be lost there, even though the individual controls are self-explanatory. 
And anyway, it's usually the case that the TW writing a document is in charge of maintaining that, so I'd leave it up to the TW. 
If you strongly insist, I'd be OK with adding it as a tip for the undecided that you can decide to follow or not.

Reply:
I don't think we should ever create Azure/AWS instructions, as long as we don't change company? 
If it's our own Kyma software, we hopefully can improve the usability so people don't get lost.

I'm not opposed to documenting navigation paths etc so that users find the correct screen. And of course, instructions must contain every step that's necessary to finish a task successfully. But the level of detail would be different for tech-savvy Kyma users than for, say, the target group of senior phones. 

The [SAP Guide](https://help.sap.com/docs/SNG_FOR_UA/e33c591ae4494a659a3f5f983c9d1161/80cf1399d10b47b482e833b6dbe9211c.html?state=DRAFT&locale=en-US) makes a good point about not stating the obvious; and frankly I think it makes our life easier if the Kyma and SAP style guides don't differ to much.

Reply:
> I don't think we should ever create Azure/AWS instructions, as long as we don't change company?
We use external software for development purposes and so sometimes we create documentation that touches that, for example, a guide on how to get logs from the above. 

We hardly ever describe the UI anyway, we do it in the case of Get Started guides, which are targeted for newbies and not only developers, and so being detailed here is justified + it's not like it's huge maintenance if it's just one place. 

Either way, clarity for the user takes priority over our convenience and maintenance load for me. 

I'm still for leaving it up to the author. I think it's a good compromise that we mention it and recommend it but not require it.

Reply:
Well, then especially if it's someone else's GUI, I would not want to provide click-level instructions. It's sad if the Azure/AWS docs aren't good and/or don't provide the instructions that we need, but I'm opposed to closing that gap in Kyma documentation.

Apart from the occasional "from/in the left navigation,", I think the level of detail in the GSG is absolutely fine. Any less detail would make the instructions useless. 

How about "> **TIP:** If possible, avoid..." ? It's softer than "Avoid...!" but not as soft as "Consider avoiding..."

Reply:
IMO, we can go with the TIP version, but being responsible for our UI, I must say that the click-level instructions make my life easier, and I believe it is the same for the users. So, tbh I'd stick to the current approach.

Reply:
Then I'd like to go with the "TIP: If possible..." version. Personally, I'm not planning to approve click-level instructions if they can be phrased more abstractly without being less helpful. 

But maybe we're meaning slightly different things here?

Reply:
For me TIP is fine. Just to make sure we're on the same page, by "click-level" I mean e.g. **Click the Add Button** :)